VikingQuiz.Api/Utilities/UserExtensions.cs

```diff
@@ -1,17 +1,33 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
 using System.Security.Claims;
-using System.Threading.Tasks;
+using System.Security.Cryptography;
+using System.Text;
 
 namespace VikingQuiz.Api.Utilities
 {
     public static class UserExtensions
     {
-        public static int GetUserId(this ClaimsPrincipal user)
+        public static int GetUserId(this IEnumerable<Claim> user)
         {
-            var strId = user.Claims.FirstOrDefault(x => x.Type == ClaimTypes.NameIdentifier).Value;
+            var strId = user.FirstOrDefault(x => x.Type == ClaimTypes.NameIdentifier).Value;
             return int.Parse(strId);
         }
+
+        public static string SHA256Encrypt(this string password)
+        {
+            SHA256 sha256 = SHA256Managed.Create();
+            byte[] bytes = Encoding.UTF8.GetBytes(password);
```

Comment:
This can also have a more meaningful name since "bytes" doesn`t tell me exactly what are those bytes for Try something like passwordAsBytes or anything similar